gen/flights/flightplan.py

```diff
@@ -985,6 +987,8 @@ def generate_aewc(self, flight: Flight) -> AwacsFlightPlan:
         start = builder.orbit(start, patrol_alt)
 
         return AwacsFlightPlan(
+            start_time=flight.flight_plan.mission_departure_time,
+            end_time=flight.flight_plan.mission_departure_time + timedelta(hours=4),
```

Comment:
Both of these should be `@property` because the values change when the TOT changes. If the player changes the TOT after creating the flight the start and end times assigned here will be wrong.

Also, `mission_departure_time` is not the takeoff time, it's the time that the aircraft _stops_ loitering.

I'd missed that we already had `mission_departure_time` before. Instead of adding `end_time`, use that. Instead of `start_time` let's name it `mission_start_time` and have it be:

```
@property
def mission_start_time(self) -> timedelta:
    return self.takeoff_time
```